api/screening/screeningRoute.js

```diff
@@ -0,0 +1,91 @@
+import express from 'express';
+import Screening from './Screening.js';
+import Movie from '../movie/Movie.js';
+// TODO: add when CinemaHall is ready import CinemaHall from '../cinemaHall/cinemaHallRoute.js';
+
+const router = express.Router();
+
+router
+  .route('/')
+  .get(async (req, res) => {
+    const screenings = await Screening.find({});
+    res.status(200).json(screenings);
+  })
+  .post(async (req, res) => {
+    const { movieId, cinemaHallId, price, startDate } = req.body;
+    const movie = await Movie.findById(movieId);
+    const cinemaHall = await CinemaHall.findById(cinemaHallId);
+    if (movie === undefined) {
+      res.status(400).json({
+        error: `Cannot find movie with id: ${req.params.id}'`,
+      });
+      return;
+    }
```

Comment:
perhaps we could think of an automation function that we could call like sendMessageIfUndefined(id, msg) {return status .... cannot find .... stg with id...}